dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBMongoDB/AzureCosmosDBMongoDBMemoryRecordMetadata.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel.Memory;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Microsoft.SemanticKernel.Connectors.AzureCosmosDBMongoDB;
+
+/// <summary>
+/// A MongoDB memory record metadata.
+/// </summary>
+#pragma warning disable CA1815 // Override equals and operator equals on value types
+public struct AzureCosmosDBMongoDBMemoryRecordMetadata
```

Comment:
Same as with AzureCosmosDBMongoDBMemoryRecord.  Can this be made internal?

Reply:
Made it internal